bindings/ios/MEGASdk.h

```diff
@@ -5019,6 +5043,81 @@ typedef NS_ENUM(NSUInteger, StorageState) {
  */
 - (void)setUploadLimitWithBpsLimit:(NSInteger)bpsLimit;
 
+/**
+ * @brief Initial step to upload a photo/video via iOS low-power background upload feature
+ *
+ * Call ensureMediaInfo() first in order prepare the library to attach file attributes
+ * that enable videos to be identified and played in the web browser.
+ *
+ * @return A pointer to an object that keeps some needed state through the process of
+ *         uploading a media file via iOS low power background uploads (or similar).
+ *         Caller takes ownership of the object.
```

Comment:
I think we can remove this line, because we don't need free the MEGABackgroundMediaUpload object in the application code layer.

Reply:
true, removed